files/zh-tw/web/css/css_box_model/mastering_margin_collapsing/index.md

```diff
@@ -83,11 +83,11 @@ slug: Web/CSS/CSS_box_model/Mastering_margin_collapsing
 
 以上這些內容都是發生在 Block-Level 的元素，設定 floating 和 absolutely positioned 的元素完全不用擔心邊界重疊的計算。
 
-## 標準文件
+## 規範
 
 {{Specifications}}
```

Comment:
not rendered in en-US, not for sure how to handle it 👀

Reply:
see https://github.com/mdn/translated-content/issues/13499#issuecomment-1573643030, let's keep it until the upstream has fix it.

Reply:
found a pr here https://github.com/mdn/content/pull/27113, maybe we can wait for this

Reply:
Removed in mdn/content#27147